JASP-Desktop/widgets/boundqmltextinput.cpp

```diff
@@ -299,19 +299,21 @@ void BoundQMLTextInput::_setFormulaOptions(std::string formula, bool valid)
 
 bool BoundQMLTextInput::_formulaResultInBounds(double result)
 {
-	double min		= getItemProperty("min").toDouble();
-	double max		= getItemProperty("max").toDouble();
-	bool inclusive	= getItemProperty("inclusive").toBool();
-
-	bool tooSmall = inclusive ? result < min : result <= min;
-	bool tooLarge = inclusive ? result > max : result >= max;
+	double min			= getItemProperty("min").toDouble();
+	double max			= getItemProperty("max").toDouble();
+	QString inclusive	= getItemProperty("inclusive").toString();
+	bool includeMin = (inclusive == "yes" || inclusive == "minOnly");
```

Comment:
ow right.
I think it makes more sense to have an `includeMin`, `includeMax`.
The idea of people having to enter random strings seems quite errorprone to me. And if we are going to have peopl enter strings we should at least force it to always be lowercase or something.